modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlAsynchronousApiTest.java

```diff
@@ -296,7 +277,7 @@ public void implicitTransactionsStates() {
         }
 
         // No new transactions through ddl.
-        assertEquals(0, txManagerInternal.finished() - txPrevCnt);
+        assertEquals(0, txManager.finished() - txPrevCnt);
```

Comment:
https://issues.apache.org/jira/browse/IGNITE-19916 probably we can close ?

Reply:
yes, but after merge the ticket only.